### PR TITLE
(#753) Add Video to Package Prefixes FAQ Doc

### LINKED ISSUE #753: Add Gary's Video on .install & .portable packages to FAQ
## What New Or Updated Would You Like To See?
Add Gary's video going over how and why we use .install and .portable packages on the CCR.

## Why Is It Needed?
It would make a good compliment to the docs on this.

## Additional Context?
NA

### PR DESCRIPTION
**Pull request by ryanrichter94**
## Description Of Changes
Added Gary's video the Chocolatey FAQ on difference between ,install and .portable packages.

## Motivation and Context
Adds additional outlet of context to the doc.

## Testing

* [X] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [X] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [X] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #753 

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Fixes #753

### CODE REVIEW

**Review by gep13 — CHANGES_REQUESTED** [review]

File: input/en-us/faqs.md [hunk 1]
```diff
@@ -186,6 +186,11 @@ Also consider that if the package **is** using `$env:ChocolateyBinRoot` (which w
 
 What is the difference between packages named *.install (i.&nbsp;e. [autohotkey.install](https://community.chocolatey.org/packages/autohotkey.install)), *.portable (i.&nbsp;e. [autohotkey.portable](https://community.chocolatey.org/packages/autohotkey.portable)) and * (i.&nbsp;e. [autohotkey](https://community.chocolatey.org/packages/autohotkey))?
 
+<div class="ratio ratio-16x9">
```
[the remark this answers is not in the document]
Comment (st3phhays) [reply input/en-us/faqs.md:189]: I also like @gep13's suggestion on the context text 😃

[the remark this answers is not in the document]
Comment (ryanrichter94) [reply input/en-us/faqs.md]: Went in and added the video under a Video Explanation sub-heading to call it out more. Also added the `mb-3` in my HTML div calling the video. Lastly added a link to a GitHub issue around the future addition of virtual packages and removed a non-working link that went to one of Rob's old sites.

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@ryanrichter94 thanks for getting this updated!

**Pull request merged**